core/state.go

```diff
@@ -91,6 +91,8 @@ type ChainState interface {
 // ChainState is an interface for getting information about the current chain state.
 type IndexedChainState interface {
 	ChainState
+	// GetIndexedOperatorState returns the IndexedOperatorState for the given block number and quorums
+	// If the quorum is not found, the quorum will be ignored and the IndexedOperatorState will be returned for the remaining quorums
```

Comment:
Can you/did you double check that there are no instances where we are assuming that this will error out if a quorum doesn't exist?

Reply:
Yes. The only two callsites are in encoding streamer (updated in this PR) and in [retrieval](https://github.com/Layr-Labs/eigenda/blob/master/clients/retrieval_client.go#L62) which doesn't assume the error condition